workspace/src/string-ops.js

```diff
@@ -2,7 +2,7 @@ export function ins(s, i, v) {
   return s.slice(0, i) + v + s.slice(i);
 }
 
-export function del(s, i, l) {
+export function del(s, i, l = 1) {
```

Comment:
I guess we don't need to specify that l=1. We can use this function like: del("01234",0,1)